ModulemdTranslationHelpers/Utils.py

```diff
@@ -1,14 +1,111 @@
 import sys
+import logging
+import gi
+import requests
+import koji
 from babel.messages import Catalog, pofile
 from datetime import datetime
 from collections import defaultdict
 
-try:
-    import gi
-    gi.require_version('Modulemd', '2.0')
-    from gi.repository import Modulemd
-except ImportError:
-    sys.exit(77)
+gi.require_version('Modulemd', '2.0')  # noqa
+from gi.repository import Modulemd
+
+
+class KojiMock:
+
+    def __init__(self, url):
+        self.session = koji.ClientSession(baseurl=url)
+
+    def listTagged(self, tag):
+        return self.session.listTagged(tag)
+
+    def getBuild(self, build_id):
+        return self.session.getBuild(build_id)
```

Comment:
Same here. What you want here is a `dict()` keyed on `build_id` that has values in the format of `getBuild()`'s return type.